src/widgets/dragAndDrop.ts

```diff
@@ -1,25 +1,28 @@
 /* Drag and drop common functionality
  */
-import * as debug from '../debug'
+import { log } from '../debug'
 
 const mime = require('mime-types')
 
-/* global FileReader alert */
-module.exports = {
-  makeDropTarget: makeDropTarget,
-  makeDraggable: makeDraggable,
-  uploadFiles: uploadFiles
-}
 // const UI = require('../index.js') // this package
-
-function makeDropTarget (ele, droppedURIHandler, droppedFileHandler) {
-  var dragoverListener = function (e) {
+/**
+ * @ignore
+ */
+class FileObject extends File {
+  lastModifiedDate?: Date
+}
+export function makeDropTarget (ele: HTMLElement, droppedURIHandler: any, droppedFileHandler: any) {
```

Comment:
I tried to use a function type for droppedURIHandler and droppedFileHandler but I couldn't get it to work with my unit tests so I just put it back to any.  Would love to hear your suggestion on this.